package.json

```diff
@@ -26,16 +26,22 @@
         "@fluid-project/eleventy-plugin-fluid": "github:fluid-project/eleventy-plugin-fluid#751e42299bde606b526ddc2183d18ad1557f6f58"
     },
     "devDependencies": {
+        "autoprefixer": "10.0.0",
+        "clean-css": "4.2.3",
+        "cross-env": "^7.0.2",
```

Comment:
should be pinned. When installing new dependencies you should be able to use the `-E, --save-exact` flag to save an exact version. I think something like the following would work. `npm install cross-env --save-dev --save-exact`